source/agora/common/TransactionPool.d

```diff
@@ -280,13 +344,13 @@ public class TransactionPool
 
     ***************************************************************************/
 
-    version (unittest) private Transaction[] take (size_t count) @safe
+    version (unittest) private const(Transaction)[] take (size_t count)
```

Comment:
Why was `@safe` removed?

Reply:
In this function, there is a `foreach` iteration over `Transaction[]`.
Since the `opApply` overload is not `@safe`, the `@safe` here has been removed to match the attribute.

Reply:
You made it templated, so safety is inferred. Add the `@safe` back and you'll see it compiles.